<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateNumbers</class>
 <widget class="QWidget" name="GenerateNumbers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QSpinBox::up-button {
    width: 30px;
    height: 30px;
    subcontrol-origin: padding;
    subcontrol-position: top right;
}

QSpinBox::down-button {
    width: 30px;
    height: 30px;
    subcontrol-origin: padding;
    subcontrol-position: bottom right;
}</string>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QComboBox::drop-down 
{
	width: 30px;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(224, 224, 224);
    border-radius: 5px;
    border: 2px solid rgb(184, 184, 184);
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(200, 200, 200);
    border: 2px solid rgb(154, 154, 154);
}</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bits to generate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>choose generator:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
